doc/source/whatsnew/v3.0.0.rst

```diff
@@ -49,6 +49,7 @@ Other enhancements
 - :meth:`Series.cummin` and :meth:`Series.cummax` now supports :class:`CategoricalDtype` (:issue:`52335`)
 - :meth:`Series.plot` now correctly handle the ``ylabel`` parameter for pie charts, allowing for explicit control over the y-axis label (:issue:`58239`)
 - Restore support for reading Stata 104-format and enable reading 103-format dta files (:issue:`58554`)
+- Support reading Stata 102-format (Stata 1) dta files (PR 58978)
```

Comment:
```suggestion
- Support reading Stata 102-format (Stata 1) dta files (:issue: `58978`)
```

Reply:
Thanks for the correction, I wasn't sure whether that worked for linking to pull requests too.